frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+"Bottom": "\u7f6e\u5e95",
+"Header cell": "\u6a19\u982d\u5132\u5b58\u683c",
+"Row group": "\u5217\u7fa4\u7d44",
+"Column group": "\u6b04\u4f4d\u7fa4\u7d44",
+"Row type": "\u884c\u7684\u985e\u578b",
+"Header": "\u6a19\u982d",
+"Body": "\u4e3b\u9ad4",
+"Footer": "\u9801\u5c3e",
+"Border color": "\u908a\u6846\u984f\u8272",
+"Insert template...": "\u63d2\u5165\u6a23\u7248...",
+"Templates": "\u6a23\u7248",
+"Template": "\u6a23\u677f",
+"Text color": "\u6587\u5b57\u984f\u8272",
+"Background color": "\u80cc\u666f\u984f\u8272",
+"Custom...": "\u81ea\u8a02",
+"Custom color": "\u81ea\u8a02\u984f\u8272",
+"No color": "No color",
+"Remove color": "\u79fb\u9664\u984f\u8272",
+"Table of Contents": "\u76ee\u9304",
+"Show blocks": "\u986f\u793a\u5340\u584a\u8cc7\u8a0a",
+"Show invisible characters": "\u986f\u793a\u96b1\u85cf\u5b57\u5143",
+"Word count": "\u8a08\u7b97\u5b57\u6578",
+"Count": "\u8a08\u7b97",
+"Document": "\u6587\u4ef6",
+"Selection": "\u9078\u9805",
+"Words": "\u5b57\u6578",
+"Words: {0}": "\u5b57\u6578\uff1a{0}",
+"{0} words": "{0} \u5b57\u5143",
+"File": "\u6a94\u6848",
+"Edit": "\u7de8\u8f2f",
+"Insert": "\u63d2\u5165",
+"View": "\u6aa2\u8996",
+"Format": "\u683c\u5f0f",
+"Table": "\u8868\u683c",
+"Tools": "\u5de5\u5177",
+"Powered by {0}": "\u7531 {0} \u63d0\u4f9b",
+"Rich Text Area. Press ALT-F9 for menu. Press ALT-F10 for toolbar. Press ALT-0 for help": "\u8c50\u5bcc\u7684\u6587\u672c\u5340\u57df\u3002\u6309ALT-F9\u524d\u5f80\u4e3b\u9078\u55ae\u3002\u6309ALT-F10\u547c\u53eb\u5de5\u5177\u6b04\u3002\u6309ALT-0\u5c0b\u6c42\u5e6b\u52a9",
+"Image title": "\u5716\u7247\u6a19\u984c",
+"Border width": "\u6846\u7dda\u5bec\u5ea6",
+"Border style": "\u6846\u7dda\u6a23\u5f0f",
+"Error": "\u932f\u8aa4",
+"Warn": "\u8b66\u544a",
+"Valid": "\u6709\u6548",
+"To open the popup, press Shift+Enter": "\u8981\u958b\u555f\u5f48\u51fa\u8996\u7a97\uff0c\u8acb\u6309Shift+Enter",
+"Rich Text Area. Press ALT-0 for help.": "\u5bcc\u6587\u672c\u5340\u57df\u3002\u8acb\u6309ALT-0\u5c0b\u6c42\u5354\u52a9\u3002",
+"System Font": "\u7cfb\u7d71\u5b57\u578b",
+"Failed to upload image: {0}": "\u7121\u6cd5\u4e0a\u50b3\u5f71\u50cf\uff1a{0}",
+"Failed to load plugin: {0} from url {1}": "\u7121\u6cd5\u4e0a\u50b3\u63d2\u4ef6\uff1a{0}\u81eaurl{1}",
+"Failed to load plugin url: {0}": "\u7121\u6cd5\u4e0a\u50b3\u63d2\u4ef6\uff1a{0}",
+"Failed to initialize plugin: {0}": "\u7121\u6cd5\u555f\u52d5\u63d2\u4ef6\uff1a{0}",
+"example": "\u7bc4\u4f8b",
+"Search": "\u641c\u7d22",
+"All": "\u5168\u90e8",
+"Currency": "\u8ca8\u5e63",
+"Text": "\u6587\u672c",
+"Quotations": "\u5f15\u7528",
+"Mathematical": "\u6578\u5b78",
+"Extended Latin": "\u62c9\u4e01\u5b57\u6bcd\u64f4\u5145",
+"Symbols": "\u7b26\u865f",
+"Arrows": "\u7bad\u982d",
+"User Defined": "\u4f7f\u7528\u8005\u5df2\u5b9a\u7fa9",
+"dollar sign": "\u7f8e\u5143\u7b26\u865f",
+"currency sign": "\u8ca8\u5e63\u7b26\u865f",
+"euro-currency sign": "\u6b50\u5143\u7b26\u865f",
+"colon sign": "\u79d1\u6717\u7b26\u865f",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687804)
```suggestion
'colon sign': "\u79d1\u6717\u7b26\u865f",
```